tendrl/commons/message.py

```diff
@@ -50,6 +50,8 @@ def __init__(self, priority, publisher, payload, job_id=None,
         else:
             self.caller = caller
         self.priority = priority
+        if not publisher:
+            publisher = NS.publisher_id
```

Comment:
So what is the problem here?
If its already there it means its already checked right?
So, if NS.publisher_id is none then when the message object is passed to Event class, it is validated there as in :
https://github.com/Tendrl/commons/blob/develop/tendrl/commons/event.py#L17->
https://github.com/Tendrl/commons/blob/develop/tendrl/commons/message.py#L69

You are probably trying to say wherever/whoever is trying to use Message and/or Event class should only go through log_utils just so that:
1. These validations are performed as the order that ensures this validation -> Event class creation accompanying Message class creation and ensuring no existence of object of Message class without it being passed to Event class -- I am not convinced of whether this is the correct way...
2. If you are saying that using log_utils, having to make 2 calls (1 to create Message object and 1 to create Event object) can be avoided, this is probably not the correct place to comment. And the scope of this PR is not that.
There probably needs to be a separate PR to do that mass change.